.drone.yml

```diff
@@ -132,7 +132,7 @@ pipeline:
       matrix:
         PREPARE_ACCEPTANCE: true
 
-  api-acceptance-tests:
+  cli-acceptance-tests:
```

Comment:
In the:
```
su-exec www-data ./run.sh --remote --config /var/www/owncloud/apps/encryption/tests/acceptance/config/behat.yml
```
It will need something like
```
--type cli
```
on the end.

(the default is ```api``)

Reply:
Ok, got it thanks.